apps/broker-service-port/src/types.ts

```diff
@@ -0,0 +1,77 @@
+
+////////////////////////////////
+// ouput type
+import { Timestamp } from "@thecointech/firestore";
+import { eTransferData } from "@thecointech/tx-gmail";
+import { DateTime } from 'luxon';
+
+export type UserAction = "Buy"|"Sell"|"Bill";
```

Comment:
All the old DB types.  No longer necessary in the new system (I can't wait to delete this code!

Reply:
aaaannnndddd deleted